src/modules/career/online-profiles.rst

```diff
@@ -14,211 +14,68 @@ things to include, as well as things to leave out of each profile.
 
+.. figure:: figures/resume-template-links.png
+   :width: 800px
+   :alt: Resume Template Links
 
-Technical Experience
-^^^^^^^^^^^^^^^^^^^^
+   Resume Template Links
 
-After listing the skills you have, you need to back up your skills with
-some experience you have gained with those skills. You can do this by
-mentioning **projects** you have completed, or by outlining
-**professional experience** that required the use of those skills. Most
-of you do not have professional experience yet, but you have completed
-projects. You will want to include projects that used the skills you
-mentioned in your technical skills section.
+You will then be directed to this page. Click *Make a copy* to make your own editable resume.
 
-::
+.. figure:: figures/copy-doc.png
+   :width: 600px
+   :alt: Copy document prompt
 
-   Project Experience:
-   Household Budgeting Web-app -- www.familybudget.com
-   A web-app that allows users to add members to their household and track incomes and expenses by categories.
-       - Java, Spring, Hibernate, Thymeleaf, MySQL, HTML, CSS, and jQuery
-       - Aside from the budget tracking features, it utilizes Spring Security to allow for secure authentication to the web server
-       - github.com/mygithubaccount/family-budget
-
-   Blogz
-   A school project completed in LC101, it allows users to signup, create, and read blog posts.
-       - Python, Flask, SQLAlchemy, Jinja, MySQL, and HTML
-       - Beyond the assigned tasks, I included a feature that allows users to write, edit, or delete their own blog posts
-       - github.com/mygithubaccount/blogz
-
-   Personal Portfolio -- myportfolio.com
-   A portfolio I am currently building. It will eventually contain links to my other projects, 
-   and document what I am currently learning.
-       - React, JavaScript, HTML, CSS, and Bootstrap
-       - I am currently learning React to incorporate into this project
-       - github.com/mygithubaccount/portfolio
-
-You need to include: - The project name - A brief description of the
-project - Skills/Technologies used in the project - A key feature of your project -
-github repo link
-
-You should include a live link to your project, if it is online.
-
-Technical Education
-^^^^^^^^^^^^^^^^^^^
+   Copy document prompt
 
-You need to show how you learned your technical skills. You can do this
-by including college courses you have taken, programs you have been a
-part of, books you have worked through, and MOOC’s you have completed.
+From there, you can pick one of two template formats to edit. Edit only the template text and NOT the formatting. Do NOT delete the LaunchCode brands on the top right. If you already have text from an older resume, you can copy and paste text into your new resume. When doing so, be sure to match destination formatting (Command+SHIFT+V for Macs and CTRL+SHIFT+V for Windows).
```

Comment:
I would add a note indicating that the second format option is on the second page